lib/Elastica/Multi/ResultSet.php

```diff
@@ -144,7 +144,7 @@ public function offsetExists($offset)
      */
     public function offsetGet($offset)
     {
-        return isset($this->_resultSets[$offset]) ? $this->_resultSets[$offset] : null;
```

Comment:
I don't think this change is related to removing support for 5.6 but an improvement because we can use these features now. As in state in my other comments, lets do such changes in separate PR.

Reply:
Okay, gonna rollback this changes, sorry.